tests/model_servering/server/llm/conftest.py

```diff
@@ -0,0 +1,177 @@
+import pytest
```

Comment:
is there any reason to add  `conftest.py` in test dir ?
imo we can add it in utils

Reply:
because placing the fixture in conftest.py is the way pytest uses them - https://docs.pytest.org/en/stable/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files

test-dir conftest files are used to contain fixtures that are only relevant for the specific test (and not to other tests)

Reply:
what if trusti AI or runtime wanted to create  a diff dir where that should add there test will they need to create a new conftest for that ?

Reply:
- tests
   - model_sering
       - server
       -  runtime
       - trusty_ai
   - <other component>
   - 

Depending on the usage of a fixture, it will located close to the tests

Reply:
umm ... still not clear to me 

- tests
   - model_sering
       - server
       -  runtime
       - trusty_ai
       - confttest.py
   is this how u ment?
   or 
   - tests
   - model_sering
       - server
            - confttest.py
       -  runtime
           - confttest.py
       - trusty_ai
          - confttest.py

Reply:
`conftest.py` can be on any level; you can have multiple files:
1. project level (pytest built-in fixtures should be palced there)
2. tests dir level
3. any sub-dir level